internal/pkg/config/matcher_test.go

```diff
@@ -0,0 +1,369 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"testing"
+
+	"github.com/google/go-flow-levee/internal/pkg/config/regexp"
+	"sigs.k8s.io/yaml"
+)
+
+func TestFuncMatcherUnmarshaling(t *testing.T) {
+	testCases := []struct {
+		desc, yaml        string
+		shouldErrorOnLoad bool
+	}{
+		{
+			desc: "Unmarshaling is strict",
+			yaml: `
+Blahblah: foo
+PackageRE: bar`,
+			shouldErrorOnLoad: false, // TODO true
```

Comment:
You are likely already aware of this, but I want to bring attention to the fact that if it weren't for this case, `shouldErrorOnLoad` would be `true` in all cases. Maybe this test should be named differently to reflect that it is testing unmarshaling failure cases (and the boolean removed).

Reply:
I'll rename / refactor in the PR that resolves this case.  Otherwise, the test case would need to be commented out or dropped to avoid breaking Travis.

So as much as I hate TODOs, I think it might be the least of the evils for our tracking of suppressed tests.